src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -40,25 +40,22 @@ trait BaggedResult extends PredictionResult[Any] {
   * @param predictions for each constituent model
   * @param NibIn       the sample matrix as (N_models x N_training)
   * @param bias        model to use for estimating bias
-  * @param repInput    representative input
   */
 case class BaggedSingleResult(
```

Comment:
Why is `BaggedSingleResult` a case class but the other two are classes?

Reply:
No good reason.  I got into case classes after the other classes were already written.